etc/kworkflow_template.config

```diff
@@ -12,6 +12,11 @@ ssh_ip=localhost
 # Default ssh port
 ssh_port=22
 
+# Optional ssh configuration file to be used
```

Comment:
Make it clear that you're talking about the config file from `ssh` (e.g., `~/.ssh/config`) because the first time I read it, I thought that you're adding a new config file kw.